tests/e2e/cucumber/features/integrations/share.ocis.feature

```diff
@@ -35,26 +33,66 @@ Feature: share
       | resource   | to               |
       | simple.pdf | folder_to_shared |
     When "Alice" opens the "files" app
-    #Then "Alice" should see the following resources
-    #  | folder_to_shared/lorem_new.txt |
-    #  | folder_to_shared/simple.pdf    |
     And "Alice" uploads the following resource
       | resource          | to               | create_version |
       | PARENT/simple.pdf | folder_to_shared | true           |
-    #Then "Alice" should see that the resource "folder_to_shared/simple.pdf" has 1 version
     And "Brian" downloads old version of the following resource
       | resource   | to               |
       | simple.pdf | folder_to_shared |
     When "Brian" restores following resources
       | resource   | to               | version |
       | simple.pdf | folder_to_shared | 1       |
-    #Then "Brian" should see that the version of resource "simple.pdf" has been restored
     When "Alice" deletes the following resources
       | resource                       |
       | folder_to_shared/lorem_new.txt |
       | folder_to_shared               |
     And "Alice" logs out
-    And "Brian" opens the "files" app
-    #Then "Brian" should not see the following resource
-    #  | Shares/folder_to_shared |
     And "Brian" logs out
+
+
+  Scenario: file
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the personal space page
```

Comment:
isn't files app also a personal space page?

Reply:
Same concern as @SwikritiT, there are other features where a similar thing is done. I think both ways take to the personal space page.

Reply:
It was copy-paste. I deleted the duplicated step `And "Alice" navigates to the personal space page`